readium/adapters/pspdfkit/pspdfkit-navigator/src/main/java/org/readium/adapters/pspdfkit/navigator/PsPdfKitDocumentFragment.kt

```diff
@@ -192,9 +192,9 @@ internal class PsPdfKitDocumentFragment(
         }
 
         private val allowedTextSelectionItems = listOf(
-            R.id.pspdf__text_selection_toolbar_item_share,
-            R.id.pspdf__text_selection_toolbar_item_copy,
-            R.id.pspdf__text_selection_toolbar_item_speak
+            com.pspdfkit.R.id.pspdf__text_selection_toolbar_item_share,
+            com.pspdfkit.R.id.pspdf__text_selection_toolbar_item_copy,
+            com.pspdfkit.R.id.pspdf__text_selection_toolbar_item_speak
```

Comment:
Does this change have consequences? Maybe these resources cannot be overridden any more.

Reply:
[This](https://github.com/readium/kotlin-toolkit/pull/368#discussion_r1273524776) is actually because android.nonTransitiveRClass defaults to true with the [updated AGP version](https://developer.android.com/build/releases/gradle-plugin#default-changes).

Another fix would be to set that to false in gradle.properties, then we can continue using `R.id.something` instead of `package.namespace.R.id.something`. There is only one other place this was necessary and it was in the testapp.

Reply:
I confirm that we can still override PSPDFKit strings with these changes, from an app's module. I tried with this:

```xml
<string name="pspdf__action_menu_copy">Test</string>
```